lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -160,68 +160,83 @@
 '''
 
 EXAMPLES = '''
-  - name: Create a new provider in ManageIQ ('Hawkular' metrics)
-    manageiq_provider:
-      name: 'EngLab'
-      type: 'OpenShift'
-      provider:
-        auth_key: 'topSecret'
-        hostname: 'example.com'
-        port: 8443
-        verify_ssl: False
-      metrics:
-        role: 'hawkular'
-        hostname: 'example.com'
-        port: 443
-        verify_ssl: False
-      manageiq_connection:
-        url: 'http://127.0.0.1:3000'
-        username: 'admin'
-        password: 'smartvm'
-        verify_ssl: False
-
-  - name: Update an existing provider named 'EngLab' (defaults to 'Prometheus' metrics)
-    manageiq_provider:
-      name: 'EngLab'
-      type: 'Openshift'
-      provider:
-        auth_key: 'verySecret'
-        hostname: 'next.example.com'
-        port: 8443
-        verify_ssl: False
-      metrics:
-        hostname: 'next.example.com'
```

Comment:
hi, @pavelzag , we just found that you need to add the "auth_key" explicitly to each endpoint, so we need the:
`auth_key: 'verySecret'` line, here, and in all the endpoints that it's not explicitly set :-(

It look like this line does not set the default auth when it is not explicitly set:
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/remote_management/manageiq/manageiq_provider.py#L376

We need to look into it, but until we do, we need to set this explicitly, sorry :-)